python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -242,10 +242,10 @@ def delete_detection(self, detection_id: str):
         raise Exception(res.text)
 
     @verify_credentials
-    def create_threat_hunt(self, test_id: str, name: str, query: str, threat_hunt_id: str = None):
+    def create_threat_hunt(self, control: Control, test_id: str, name: str, query: str, threat_hunt_id: str = None):
```

Comment:
do we consider this a breaking change?

Reply:
i would say it's a breaking change but is this even being used anywhere?

Reply:
technically a breaking change, but I doubt it's being used yet so maybe something we can document and proceed. Otherwise we have to default it to crowdstrike which feels wrong